dm/tests/all_mode/data/db3.prepare.sql

```diff
@@ -0,0 +1,42 @@
+drop database if exists `all_mode`;
+create database `all_mode`;
+use `all_mode`;
+
+create table t_extract
+(
+    id int primary key auto_increment,
+    j  json,
+    a  int as (j ->> '$.a'),
+    b  char(10) as (j ->> '$.b')
+);
+
+insert into t_extract(j)values ('{"a":1, "b": {"a":1}}');
+insert into t_extract(j)values ('{"a":1, "b": [1,2,3]}');
+insert into t_extract(j)values ('[1,2]');
+insert into t_extract(j)values ('true');
+insert into t_extract(j)values ('false');
+insert into t_extract(j)values (null);
+insert into t_extract(j)values ('"string"');
+insert into t_extract(j)values ('1');
+insert into t_extract(j)values ('3.14');
+insert into t_extract(j)values ('{"a":1, "b": {"a":1}, "c": "string"}');
+
+create table t_funcs
+(
+    id int primary key auto_increment,
+    j  json,
+    j1 json as (json_extract(j, '$.a')),
+    j2 json as (json_extract(j1, '$.b')),
```

Comment:
Does it cover all valid JSON function which can be used as generating expression?

Reply:
it covered now